### ISSUE TITLE
Recording and arming on midi input

### ISSUE THREAD

**Issue by miniukof** [monocasual/giada#124]
Setting channels midi input for arm does not work, midi input is set but pressing doesn't do anything.
After setting global midi input recording and sending that input giada crashes with:
```
 >>> inputRec (master) (pure=0x902F0000)
X I/O error
```

Tested on linux with version 0.13.2 from tar and also compiled from git: 53be5fc20d.
Tried with both jack and alsa midi inputs, same outcome.

**Comment by monocasual** [comment]
thanks for reporting @miniukof . Could you please paste here the console output when you send the MIDI event to Giada? I suppose you should see something like `[KM] MIDI received - [...]` at least...

**Comment by miniukof** [comment]
Full output.
I created sample channel, set midi input for arm then tried pressing it few times, nothing happened.
Then I set midi input for start recording send that input and it crashed.

```
[init] Giada 0.13.2 - Fri Mar 17 19:06:23 2017
[init] configuration file ready
[MidiMapConf::init] scanning midimaps directory...
[MidiMapConf::init] unable to scan midimaps directory!
[MidiMapConf::read] midimap not specified, nothing to do
[init_prepareMidiMap] JSON-based midimap read failed, trying with the deprecated one...
[MidiMapConf::readMap_DEPR_] midimap not specified, nothing to do
[KA] using system 0x1
[KA] Opening devices 0 (out), 0 (in), f=48000...
[KA] 5 device(s) found
  0) system
  1) deadbeef
  2) system
  3) PulseAudio JACK Sink-01
  4) PulseAudio JACK Source-01
[KA] JACK in use, freq = 48000
[PluginHost::init] initialize with buffersize=128, samplerate=48000
[KM] using system 0x2
[KM] 2 output MIDI ports found
  0) Midi Through 14:0
  1) MPKmini2 28:0
[KM] MIDI out port 1 open
[KM] 3 input MIDI ports found
  0) Midi Through 14:0
  1) MPKmini2 28:0
  2) Giada MIDI Output 129:0
[KM] MIDI in port 1 open
[geKeyboard::__cb_addColumn] new column added (index=0, w=380), total count=1, addColumn(x)=404
[geKeyboard::__cb_addColumn] new column added (index=1, w=380), total count=2, addColumn(x)=800
[geKeyboard::__cb_addColumn] new column added (index=2, w=380), total count=3, addColumn(x)=1196
[geKeyboard::__cb_addColumn] new column added (index=3, w=380), total count=4, addColumn(x)=1592
[geKeyboard::__cb_addColumn] new column added (index=4, w=380), total count=5, addColumn(x)=1988
[geKeyboard::__cb_addColumn] new column added (index=5, w=380), total count=6, addColumn(x)=2384
[KA] Jack transport stopped, frame=0
[KA] latency = 512
[geColumn::__cb_addChannel] index = 0
[mixer] channel index=0 added, type=1, total=1
[geKeyboard::addChannel] add to column with index = 0
[KM] MIDI received - 0x90334900 (chan 0)
[gdMidiGrabber] MIDI learn done - message=0x90330000
[KM] MIDI received - 0x80330000 (chan 0)
[KM] MIDI received - 0x90334900 (chan 0)
[KM] MIDI received - 0x80330000 (chan 0)
[KM] MIDI received - 0x90336300 (chan 0)
[KM] MIDI received - 0x80330000 (chan 0)
[KM] MIDI received - 0x90336C00 (chan 0)
[KM] MIDI received - 0x80330000 (chan 0)
[KM] MIDI received - 0x902F5000 (chan 0)
[gdMidiGrabber] MIDI learn done - message=0x902F0000
[KM] MIDI received - 0x802F0000 (chan 0)
[KM] MIDI received - 0x902F3B00 (chan 0)
  >>> inputRec (master) (pure=0x902F0000)
```

**Comment by monocasual** [comment]
Should be fixed in 0.14.4. Feel free to re-open it otherwise.
